### ISSUE TITLE
PyAssimp crashes with glTF2 test-file

### ISSUE THREAD

**Issue by umlaeute** [assimp/assimp#2822]
after fixing #2788, i still get a crash when loading the `test/models/glTF2/glTF-Sample-Models/AnimatedMorphCube-glTF/AnimatedMorphCube.gltf` file:

~~~sh
$ gdb --args python3 -c 'import pyassimp;pyassimp.load("test/models/glTF2/glTF-Sample-Models/AnimatedMorphCube-glTF/AnimatedMorphCube.gltf", processing=None)'
(gdb) run
Starting program: /usr/bin/python3 -c import\ pyassimp\;pyassimp.load\(\"test/models/glTF2/glTF-Sample-Models/AnimatedMorphCube-glTF/AnimatedMorphCube.gltf\",\ processing=None\)
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".

Program received signal SIGSEGV, Segmentation fault.
f_get (ptr=0x1b1b1b1b1b1b1b1b, size=4) at ./Modules/_ctypes/cfield.c:1071
1071	./Modules/_ctypes/cfield.c: Datei oder Verzeichnis nicht gefunden.
(gdb) py-bt
Traceback (most recent call first):
  <built-in method getattr of module object at remote 0x7ffff78d6d10>
  File "<<ASSIMP>>/port/PyAssimp/pyassimp/core.py", line 133, in _init
    obj = getattr(self, m)
  File "<<ASSIMP>>/port/PyAssimp/pyassimp/core.py", line 80, in call_init
    _init(obj.contents, obj, caller)
  File "<<ASSIMP>>/port/PyAssimp/pyassimp/core.py", line 215, in _init
    call_init(obj, target)
  File "<<ASSIMP>>/port/PyAssimp/pyassimp/core.py", line 80, in call_init
    _init(obj.contents, obj, caller)
  File "<<ASSIMP>>/port/PyAssimp/pyassimp/core.py", line 187, in _init
    call_init(e, target)
  File "<<ASSIMP>>/port/PyAssimp/pyassimp/core.py", line 80, in call_init
    _init(obj.contents, obj, caller)
  File "<<ASSIMP>>/port/PyAssimp/pyassimp/core.py", line 187, in _init
    call_init(e, target)
  File "<<ASSIMP>>/port/PyAssimp/pyassimp/core.py", line 320, in load
    scene = _init(model.contents)
  File "<string>", line 1, in <module>
(gdb) py-up
#8 Frame 0xdbae60, for file /home/zmoelnig/src/3rdparty/assimp5/port/PyAssimp/pyassimp/core.py, line 134, in _init (self=<Vector3D() at remote 0x7ffff51fe170>, target=<LP_Vector3D at remote 0x7ffff51fe200>, parent=<LP_AnimMesh(bitangents=<...>) at remote 0x7ffff51fe050>, dirself=['__class__', '__ctypes_from_outparam__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__setstate__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__', '_b_base_', '_b_needsfree_', '_fields_', '_objects', 'x', 'y', 'z'], m='x', name='')
    obj = getattr(self, m)
(gdb) 
~~~

**Comment by umlaeute** [comment]
btw, `assimp info` can load the file just fine

**Comment by umlaeute** [comment]
turns out that the PyAssimp-wrapper missed the `mName` member of the `AnimMesh` structure.
this is also the reason why `test/models/AMF/test_with_mat.amf` failed to load in #2821 (although it didn't crash).

for whatever reasons the struct-mismatch didn't trigger crashes in the other test-models.

The fix is included in #2813
